packages/clay-pagination/src/Pagination.tsx

```diff
@@ -10,24 +10,45 @@ import Ellipsis from './Ellipsis';
 import Item from './Item';
 
 interface IProps extends React.HTMLAttributes<HTMLUListElement> {
+	/**
+	 * Use this property for defining `otherProps` that will be passed to the `nav` element.
+	 */
+	navProps?: {
+		'aria-label'?: string;
+		role?: string;
+	};
```

Comment:
I'm not sure of the `navProps` API in fact I want us to avoid APIs like this for components, this is a common prop drilling issue in React.js, these are usually signs that we don't have good compositing or something it's wrong.

Considering this component's scenario, we can avoid `navProps` and we don't need to leave the `role` configurable, I don't think we have another alternative for this component right?! then keep the `...otherProps` in the new `nav` container instead of the list.

Reply:
Wouldn't this be a breaking change? We've had otherProps mapped to the `ul` for a long time.

Reply:
Yes, in theory that would be a significant change, but I'd rather do that than add a new API specific to it, if we think that adding `nav` is also considering a bug that the component didn't have, it's not a breaking change. In theory the `aria-label` will be in the right place, in the `nav` instead of the `ul` and just like role for example, what else could affect would be the `className`. It makes sense?

Reply:
I ended up mapping `aria-label` to the `nav` and not applying `role="navigation"`. I looked at several pagination examples and some don't include it. I went with [Mozilla's example](https://developer.mozilla.org/en-US/docs/Web/CSS/Layout_cookbook/Pagination). We can add it later if it's needed.

I think `...otherProps` is better mapped to the `ul` element. All the modifier classes are added on that element and moving it to `nav` would cause issues.

Reply:
Makes sense @pat270, looks good to me!